src/properties.ts

```diff
@@ -42,7 +43,6 @@ function getDefaultProperties(): ScannerProperties {
       process.env.HOME ?? process.env.USERPROFILE ?? '',
       SONAR_DIR_DEFAULT,
     ),
-    [ScannerProperty.SonarScannerCliVersion]: SCANNER_CLI_VERSION,
```

Comment:
As discussed, dropped this from default props

Reply:
you can drop this from the deps above
https://github.com/SonarSource/sonar-scanner-npm/pull/128/files#diff-c0e4912587622579192cb35813490cac16223f5edb98b8e72d7aed68fc8ffc11L29